reference/conanfile/running_and_output.rst

```diff
@@ -39,7 +39,7 @@ Running commands
 
 .. code-block:: python
 
-    run(self, command, stdout=None, cwd=None, ignore_errors=False, env="", quiet=False, shell=True, scope="build")
+    run(self, command, stdout=None, cwd=None, ignore_errors=False, env="", quiet=False, shell=True, scope="build", stderr=None)
```

Comment:
I think we don't have any code snippet to show how to capture the stdout or stderr in the recipe, maybe this would be a good opportunity to add some code to explain how to capture the ouput.

Reply:
👍 
Good idea

Reply:
Done, please check